opal_server/pubsub.py

```diff
@@ -25,6 +29,8 @@ def __init__(self, signer: JWTSigner, broadcaster_uri: str = None):
         self.router = APIRouter()
         # Pub/Sub Internals
         self.notifier = WebSocketRpcEventNotifier()
+        self.notifier.add_subscription_restriction(self._verify_permitted_topics)
+
```

Comment:
@idan-ben-ami i think you changed the name of the function to `add_channel_restriction`.

can you please fix it?

Reply:
also: the tests will continue failing: you need to fix the `requirements.txt` file of OPAL to require fastapi_websocket_pubsub to be at least 0.2.0 so that these new API be available to you.